11-getting-started-with-Jupyter.Rmd

```diff
@@ -0,0 +1,311 @@
+
+The name Jupyter came from combining the names of the three programming languages that it was initially targeted for (Julia, Python, and R), and now many other languages can be used with Jupyter notebooks.
+
+An example of what a Jupyter notebook looks like is shown in \@ref(fig:img-jupyter).
+
+```{r img-jupyter, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "A screenshot of a Jupyter Notebook.", fig.retina = 2}
+knitr::include_graphics("img/jupyter.png")
+```
+
+### Accessing Jupyter
+
+One of the easiest ways to start computing in R with Jupyter is to use a 
+web-based platform called JupyterHub that already has Jupyter, R, a number of R packages, and collaboration tools installed, configured and ready to use. 
+JupyterHub's are usually created and provisioned by organizations (for example,
+a university, a company, *etcetera*) and require authentication to gain 
+access. <!--- Insert link to public JupyterHub here if we can get 
+access/permission (either 2i2c collaboration, or 
+https://notebooks.gesis.org/hub/home) --> Jupyter can also be installed on your
+own computer (desktop or laptop), and we provide examples of how to do this in
+chapter \@ref(move-to-your-own-machine).
+
+## Code cells
+
+The sections of a Jupyter notebook that contain code in the R programming 
+language are referred to as code cells. A code cell that has not yet been executed 
+has no number inside the square brackets to the left of the cell 
+(Figure \@ref(fig:code-cell-not-run)). Running a code cell will execute all of 
+the code it contains, and the output (if any exists) will be displayed directly
+underneath the code that generated it. Outputs may include printed text or 
+numbers, data frames and data visualizations. Cells that have been executed 
+also have a number inside the square brackets to the left of the cell. 
+This number indicates the order in which the cells were run 
+(Figure \@ref(fig:code-cell-run)).
+
+```{r code-cell-not-run, echo = FALSE, fig.cap = "A code cell in Jupyter that has not yet been executed. Note the absence of a number inside the square brackets to the left of the cell.", fig.retina = 2, out.width="100%"}
+image_read("img/code-cell-not-run.png") %>% 
+  image_crop("3632x1000")
+```
+
+```{r code-cell-run, echo = FALSE, fig.cap = "A code cell in Jupyter that has been executed. Note the code output beneath the cell (preview of a data frame) and the presence of a number inside the square brackets to the left of the cell. This number indicates the order in which the cells were run.", fig.retina = 2, out.width="100%"}
```

Comment:
```suggestion
```{r code-cell-run, echo = FALSE, fig.cap = "A code cell in Jupyter that has been executed.", fig.retina = 2, out.width="100%"}
```